debian_slim/Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM debian:jessie
+ARG CLOUD_SDK_VERSION=160.0.0
```

Comment:
we have to add in the version directly..thats the only way the tagged release from github can be consistent. (i,e if the version isn't in the dockerfile, if i pull the source for tag version X and build it, it'd use 'latest' otherwise.